app/views/courses/_status_panel.html.erb

```diff
@@ -8,17 +8,30 @@
     <%= course.on_find %>
   </p>
 
-  <h3 class="govuk-heading-s govuk-!-margin-bottom-0">Applications</h3>
-  <p class="govuk-body" data-qa="course__open_for_applications"><%= course.open_or_closed_for_applications %></p>
+  <% if course.is_published? %>
+    <h3 class="govuk-heading-s govuk-!-margin-bottom-0">Applications</h3>
+    <p class="govuk-body" data-qa="course__open_for_applications"><%= course.open_or_closed_for_applications %></p>
 
-  <h3 class="govuk-heading-s govuk-!-margin-0">Vacancies</h3>
-  <p class="govuk-body" data-qa="course__has_vacancies">
-    <%= course.vacancies %>
-  </p>
+    <h3 class="govuk-heading-s govuk-!-margin-0">Vacancies</h3>
+    <p class="govuk-body" data-qa="course__has_vacancies">
+      <%= course.vacancies %>
+    </p>
 
-  <% if course.last_published_at.present? %>
+    <% if course.last_published_at.present? %>
+      <hr class="govuk-section-break govuk-section-break--visible govuk-section-break--m">
+      <p class="govuk-body" data-qa="course__last_published_date">
+        Last published:<br>
+        <%= l(course.last_published_at&.to_date) %>
+      </p>
+    <% end %>
+  <% else %>
     <hr class="govuk-section-break govuk-section-break--visible govuk-section-break--m">
```

Comment:
I think the status panel would be easier to parse if this block were in its own partial.

Reply:
Any easier to read now @benilovj ?